src/middleware/router.js

```diff
@@ -13,10 +13,19 @@ import * as utils from '../utils'
 import * as messageStore from '../middleware/messageStore'
 import * as events from '../middleware/events'
 import { promisify } from 'util'
+import { getGridFSBucket } from '../contentChunk'
+import { Writable, Readable } from 'stream';
+import util from 'util'
+import mongodb from 'mongodb'
+import { connectionDefault } from '../config'
+import { brotliCompressSync } from 'zlib';
```

Comment:
Not sure what this is used for? We shouldn't include modules and functions that arent being used